tsdb/head_test.go

```diff
@@ -1810,3 +1810,105 @@ func newTestHead(t testing.TB, chunkRange int64, compressWAL bool) (*Head, *wal.
 		testutil.Ok(t, os.RemoveAll(dir))
 	}
 }
+
+func TestHeadLabelNamesValuesWithMinMaxRange(t *testing.T) {
+	head, _, closer := newTestHead(t, 1000, false)
+	defer closer()
+	defer func() {
+		testutil.Ok(t, head.Close())
+	}()
+
+	const (
+		firstSeriesTimestamp  int64 = 100
+		secondSeriesTimestamp int64 = 200
+		lastSeriesTimestamp   int64 = 300
+	)
+	var (
+		seriesTimestamps = []int64{firstSeriesTimestamp,
+			secondSeriesTimestamp,
+			lastSeriesTimestamp,
+		}
+		expectedLabelNames  = []string{"a", "b", "c"}
+		expectedLabelValues = []string{"d", "e", "f"}
+	)
+
+	app := head.Appender()
+	for i, name := range expectedLabelNames {
+		_, err := app.Add(labels.Labels{{Name: name, Value: expectedLabelValues[i]}}, seriesTimestamps[i], 0)
+		testutil.Ok(t, err)
+	}
+	err := app.Commit()
+	testutil.Ok(t, err)
```

Comment:
We can inline the above to: 
```suggestion
	testutil.Ok(t, app.Commit())
```

Reply:
(Goal: Bit less code to read without complexity, so less cognitive load)